packages/itwin/property-grid/src/types.ts

```diff
@@ -56,13 +57,13 @@ export interface PropertyGridProps {
   rulesetId?: string;
   rootClassName?: string;
   dataProvider?: PresentationPropertyDataProvider;
-  onInfoButton?: () => void;
   onBackButton?: () => void;
```

Comment:
do you want to kill off the onBackButton too? 
Should that be added via the headerContent?

Reply:
That's for a future change...  The onBackButton prop is used in the multi-selected element state, and I thought that was tackling a bit more than I like for this PR.